index.html

```diff
@@ -812,152 +812,200 @@ <h2>Privacy Considerations</h2>
       </p>
     </section>
 
-    <section class="appendix">
+    <section class="appendix" id="TestVectors">
       <h2>Test Vectors</h2>
       <p>
-The following test vectors are provided to assist with implementers.
+        For all representations the signer needs to generate a private/public key pair with the private key used 
+        for signing and the public key made available for verification. The 
+        [[MULTIBASE]]/[[MULTICODEC]] representation for the public key, <code>ed25519-pub</code>,
+        and the representation for the private key, <code>ed25519-priv</code>, are shown below.
       </p>
+      <pre class="example" title="Private and Public keys for Signature"
+          data-include="TestVectors/keyPair.json" data-include-format="text"></pre>
+      
+      <section id="VectDataIntegrity">
```

Comment:
No need to specify `id` values most of the time, Respec generates them automatically.

If you do hand-code an `id` value, use all lowercase letters, numbers, and dashes only (or we might fail publication rules checking at W3C ... specifically the anchoring checks when we go to publish to W3C). Let the tools do the heavy lifting for us and auto-generate `id`.

Reply:
Will do. I put the `id` value in to help navigate in my IDE (VS code) otherwise it gets hard to find things in the raw HTML. If you have other tricks let me know.